h2o-py/h2o/h2o.py

```diff
@@ -1405,6 +1405,31 @@ def save_model(model, path="", force=False):
     return api("GET /99/Models.bin/%s" % model.model_id, data={"dir": path, "force": force})["dir"]
 
 
+def download_model(model, path=""):
```

Comment:
I think for save we have a convenience method model.save() we should have one for download as well I think